src/main/app/common/country.ts

```diff
@@ -0,0 +1,122 @@
+import { Form, FormValidationError } from 'forms/form'
```

Comment:
This file has too many different constructs that fulfil so many different responsibilities. That gives the file too many reasons for change. Please split responsibilities in better way.